content-based-routing/README.md

```diff
@@ -0,0 +1,52 @@
+# Content Based Routing
```

Comment:
Can you explain in a section what is different about the setup compared to the quick start? This will help point out what is diff required to setup content-based routing.

Reply:
Done